flowpipe/plug.py

```diff
@@ -14,6 +14,20 @@
 if sys.version_info.major > 2:  # pragma: no cover
     basestring = str
 
+class IterationPlug(list):
```

Comment:
If it's not a plug, we shouldn't name it like that. How about `IterationTrigger` or `IterationList`?

Reply:
Sure, thanks for your extensive feedback. Besides the PR is a WIP, happy to change it in a way that it fits best for the repo.